src/sync/sys/windows/net.rs

```diff
@@ -359,4 +383,76 @@ mod test {
             }
         }
     }
+
+    #[test]
+    fn should_accept_new_client() {
+        let listener = Arc::new(PipeListener::new(r"\\.\pipe\ttrpc-test-accept").unwrap());
```

Comment:
Suggest defining the `r"\\.\pipe\ttrpc-test-accept"` as a const, for the variable used again in line 407.